docs/liveliness.rst

```diff
@@ -12,13 +12,10 @@ duration as long as the DomainParticipant keeps running and remote participants
 entities within the DomainParticipant alive.
 
 The remaining kinds (MANUAL_BY_PARTICIPANT and MANUAL_BY_TOPIC) need periodically an assertion to consider the remote 
-participants as alive. That assertion can be done by writing new data or using the function *assert_liveliness*. The 
-main difference between these two kinds is that MANUAL_BY_PARTICIPANT requires only the assertion of one of its 
-entities to consider that all of them are alive, while MANUAL_BY_TOPIC requires at least one assertion per publisher 
-to consider it as alive.
```

Comment:
Why did you remove this explanation? Did you consider it difficult to understand?

Reply:
I thought that explanation was too technical and since the test did not use any of those kinds, I decided to remove it. But if you think it can be understood correctly, I can add it again.

Reply:
I think removing it is fine, I was just asking :)